src/app/api/routers/v1/tasks.py

```diff
@@ -1,42 +1,37 @@
-import logging  # noqa
-from typing import Annotated  # noqa
+import logging
 from uuid import UUID
 
-from fastapi import APIRouter, Depends, status  # noqa
+from fastapi import APIRouter, status
 
-from app.api.dependencies.repositories import TaskCommentRepositoryDep  # noqa
-from app.api.dependencies.services import create_tasks_service
+from app.api.dependencies.repositories import TaskCommentRepositoryDep, TaskRepositoryDep
 from app.schemas.comment import CreateTaskCommentSchema, GetTaskCommentSchema
 from app.schemas.tasks import TaskCreateSchema, TaskGetSchema, TaskUpdateSchema
-from app.services.tasks import TasksService
 
 router = APIRouter(prefix="/tasks", tags=["Tasks"])
 
 logger = logging.getLogger().getChild("task-router")
 
-TasksServiceDep = Annotated[TasksService, Depends(create_tasks_service)]
-
 
 @router.get("", summary="Получить задачу по id", response_model=TaskGetSchema, status_code=status.HTTP_200_OK)
-async def _get_task(task_id: UUID, tasks_service: TasksServiceDep) -> TaskGetSchema:
+async def _get_task(task_id: UUID, tasks_service: TaskRepositoryDep) -> TaskGetSchema:
```

Comment:
```suggestion
async def _get_task(task_id: UUID, tasks_repository: TaskRepositoryDep) -> TaskGetSchema:
```

и в других местах тоже